Formula/ldapvi.rb

```diff
@@ -45,6 +45,9 @@ class Ldapvi < Formula
   end
 
   def install
+    # Workaround for Xcode 14.3.
+    ENV.append_to_cflags "-Wno-implicit-function-declaration"
```

Comment:
```suggestion
    ENV.append_to_cflags "-Wno-implicit-function-declaration" if DevelopmentTools.clang_build_version == 1403
```
This should technically be reported and fixed upstream, so we should do something like this so it nudges us to deal with it properly later.